markdown/3.0/en/ff-paging.md

```diff
@@ -18,7 +18,7 @@ You can access the value of a paging item with the ** {{caption}}**  syntax.
 
 When the attribute `show-only` is set to ** true**  the paging does not react on clicks.
```

Comment:
`true` is probably be meant to be bold. Maybe we should fix this while we are at it.

Reply:
as above